search/elastic.py

```diff
@@ -209,6 +209,7 @@ def log_indexing_error(cls, indexing_errors):
         """ Logs indexing errors and raises a general ElasticSearch Exception"""
         indexing_errors_log = []
         for indexing_error in indexing_errors:
+            print "error: {}".format(indexing_error)
```

Comment:
debugging, revealed `indexing_error` to be formatted as `{u'index': {u'status': 201, u'_type': u'courseware_content', u'_id': u'block-v1:edX+DemoX+Demo_Course+type@html+block@030e35c4756a4ddc8d40b95fbbfff4d4', u'_version': 1, u'_index': u'courseware_index'}}`